net/ser2net/Makefile

```diff
@@ -24,11 +24,16 @@ PKG_INSTALL:=1
 
 include $(INCLUDE_DIR)/package.mk
 
+CONFIGURE_ARGS += \
```

Comment:
It seems you are doing more changes than just "update to version 4.3.5" as said in the commit message.

Reply:
I just added that for consistency, so that it always compiled with the same settings, instead of leaving configure to guess.